### PR TITLE
feat(rest): adding additional fields in project doc.

### LINKED ISSUE #2437: Adding additional fields in project doc.
### Description

Adding additional fields like releaseName and version in releaseIdToUsage field in project document.
Also adding projectOwner field in REST API project endpoint.

### PR DESCRIPTION
**Pull request by rudra-superrr**
Isssue: Closes #2437

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -103,7 +103,7 @@
     <cloudant.version>2.19.1</cloudant.version>
     <commons-codec.version>1.15</commons-codec.version>
     <commons-collection4.version>4.4</commons-collection4.version>
-    <commons-compress.version>1.26.0</commons-compress.version>
+    <commons-compress.version>1.21</commons-compress.version>
```
Comment (GMishx) [inline pom.xml:106]: Please revert the changes.

File: frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/portlets/components/ComponentPortlet.java
Comment (GMishx) [inline frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/portlets/components/ComponentPortlet.java:1]: Please remove changes from front-end.

**Review by GMishx — CHANGES_REQUESTED** [review]
Please rebase the branch with latest main.

**Comment by rudra-superrr** [comment]
Note: On updating release version from UI, the newly added field release version should also get updated in projectReleaseRelationship field in project document. This functionality should be added in new UI.

**Comment by GMishx** [comment]
> Note: On updating release version from UI, the newly added field release version should also get updated in projectReleaseRelationship field in project document. This functionality should be added in new UI.

Please open an issue in the https://github.com/eclipse-sw360/sw360-frontend/ project referencing this PR so it does not get missed.

Also, the test cases are failing for the PR. Please check.

**Comment by GMishx** [comment]
Required data is available over `/releases` endpoint. Thus closing the PR.

**Pull request closed without merging**